news-aggregation/NewsAggregationApp.ts

```diff
@@ -32,6 +44,19 @@ export class NewsAggregationApp extends App {
 		await sendDirectMessageOnInstall(read, modify, user, persistence);
 	}
 
+	public async onEnable(
+		environment: IEnvironmentRead,
+		configurationModify: IConfigurationModify
+	): Promise<boolean> {
+		await Promise.all([
+			configurationModify.scheduler.scheduleRecurring({
+				id: 'fetch-news',
+				interval: '* * * * * *',
```

Comment:
```suggestion
				interval: '* * * * *',
```
Beware of bombarding the news source API every second, their servers can block our access. Plus this also floods the log. For testing, an interval of atleast 10 seconds are fine.